openpype/hosts/houdini/plugins/publish/validate_sop_output_node.py

```diff
@@ -55,6 +57,16 @@ def get_invalid(cls, instance):
             )
             return [output_node.path()]
 
+        # Output node must be a Sop Output node.
+        if not output_node.type().name() == "output":
+            cls.log.error(
+                "Output node %s is not a SOP Output node. "
+                "SOP Path must point to a SOP Output node, "
+                "instead found sop type: '%s'"
+                % (output_node.path(), output_node.type().name())
+            )
+            return [output_node.path()]
```

Comment:
I'd say this is not a requirement. All that is needed is that it's a SOP node. Doesn't have to be an output node.
```suggestion
```

Or why do you think it **must** be an output node for the publish to work?